File: hsqldb/jython/read/hsqldb_read.py
#! /usr/bin/jython
# -*- coding: utf-8 -*-
#
#	hsqldb_read.py
#					Nov/29/2010
#
# ----------------------------------------------------------------
#
import sys
import	string
from java.lang import	*
#
import java
from java.sql import *
from java.io  import *
#
# ----------------------------------------------------------------
sys.path.append ('/var/www/data_base/common/jython_common')
from jython_rdb_manipulate import display_proc
# ----------------------------------------------------------------
System.out.println ("*** start ***")
print ("*** 開始 ***")

java.lang.Class.forName("org.hsqldb.jdbcDriver")

#
str_connect = "jdbc:hsqldb:file:/var/tmp/hsqldb/cities;shutdown=true"
#
conn = DriverManager.getConnection(str_connect,"SA", "")

display_proc (conn)
#
conn.close()
print ("*** 終了 ***")
#
# ----------------------------------------------------------------

File: hsqldb/jruby/read/hsqldb_read.rb
#! /usr/bin/jruby
# -*- coding: utf-8 -*-
#
#	read/hsqldb_read.rb
#					Jan/29/2015
#
# --------------------------------------------------------
include Java
#
import java.sql.DriverManager
import java.lang.System
import java.lang.Class
#
load '/var/www/data_base/common/jruby_common/jruby_sql_manipulate.rb'
# --------------------------------------------------------
puts "*** 開始 ***"
#

str_connect = "jdbc:hsqldb:file:/var/tmp/hsqldb/cities;shutdown=true"

conn = DriverManager.getConnection(str_connect,"SA", "")

sql_read_proc(conn)

conn.close()

puts "*** 終了 ***"
# --------------------------------------------------------
